bukkit/src/main/java/net/vyhub/tasks/TStatistics.java

```diff
@@ -0,0 +1,86 @@
+package net.vyhub.tasks;
+
+import net.vyhub.VyHubPlatform;
+import net.vyhub.abstractClasses.AServer;
+import net.vyhub.abstractClasses.AStatistics;
+import net.vyhub.abstractClasses.AUser;
+import net.vyhub.entity.VyHubUser;
+import net.vyhub.lib.Utility;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import retrofit2.Response;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+
+import static java.util.logging.Level.INFO;
+import static net.vyhub.lib.Utility.checkResponse;
+
+public class TStatistics extends AStatistics {
+    public TStatistics(VyHubPlatform platform, AUser aUser) {
+        super(platform, aUser);
+    }
+
+    public void sendPlayerTime() {
+        super.getPlatform().log(INFO, "Sending playertime to API");
+
```

Comment:
Maybe put this function to common

Reply:
Das stimmt! Mega guter Punkt. Die hatte nichts mit Bukkit zu tun..